### PR TITLE
Added market data for Lübeck

### PR DESCRIPTION
**Pull request by jbein**

### CODE REVIEW

**Comment by johnjohndoe** [comment]
@jbein The market data contain a few mistakes. Please have a look [here](https://travis-ci.org/wo-ist-markt/wo-ist-markt.github.io/builds/548527135?utm_source=github_status&utm_medium=notification)

File: cities/luebeck.json [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+                    53.912991
+                ]
+            }
+        }
+
+    ],
+    "metadata" : {
+        "data_source" : {
+            "url" : "https://www.luebeck-tourismus.de/einkaufen/shoppingparadies/maerkte-regionale-produkte.html",
+            "title" : "Lübeck Tourismus"
```
Comment (johnjohndoe) [inline cities/luebeck.json]: Please consider adding a time stamp so users know when the data has been updated.
Example: [berlin.json](https://github.com/wo-ist-markt/wo-ist-markt.github.io/blob/master/cities/berlin.json?short_path=7185eea#L2390)

Comment (jbein) [reply cities/luebeck.json]: Timestamp is added and errors should also be fixed.

Comment (johnjohndoe) [reply cities/luebeck.json]: :+1: Nice. Thanks.

**Comment by johnjohndoe** [comment]
@jbein, thank you once again for adding yet another city! As you already know this website is **nothing without its users**. Therefore I kindly ask you to advertise the website as best as you can. There are a few options that are easy to start with:

- [ ] Tweet it out - don't forget a nice screenshot :bird: 
- [ ] Ask the city of Lübeck if they put a link and maybe a screenshot on their website. Some cities did that and one can clearly see the user increase in the stats! :cityscape: 
- [ ] Share the new with local newspapers :newspaper: 
- [ ] Print stickers and share them: the assets can be found in the [wo-ist-markt-media repository](https://github.com/wo-ist-markt/wo-ist-markt-media) :printer: 
- [ ] Become a living advertisement - print your own t-shirt - I did that! Same assets as above :shirt: 

Of course, if you come up with something ever cooler - please let me know!

**Comment by jbein** [comment]
> Travemünde is not exactly Lübeck ... but I think its fine - people from Lübeck might really love to get fresh fish over there?!

In fact Travemünde is part of Lübeck. But i did not know that either until I was looking to the wiki page  ;o)

**Comment by johnjohndoe** [comment]
> In fact Travemünde is part of Lübeck.

Interesting. Now Wo ist Markt? becomes an educational project it seems :woman_teacher: Thank you for the clarification.

**Pull request merged**